src/bundle/Resources/public/scss/_main-row.scss

```diff
@@ -0,0 +1,68 @@
+.ez-content-view,
+.ez-bookmark-list-view,
+.ez-trash-list-view,
+.ez-drafts-list-view {
+    .ez-main-row {
+        display: grid;
+        grid-template-columns: calculateRem(80px) 1fr calculateRem(80px);
+        grid-template-rows: auto;
+        grid-template-areas: "ez-side-menu ez-content-view-container ez-context-menu";
+
+        .ez-side-menu {
+            grid-area: ez-side-menu;
+        }
+
+        .ez-content-view-container {
```

Comment:
I would combine `ez-content-view-container` and `ez-content-edit-container` into one `ez-content-container`.

Reply:
Ok!